elements/lisk-p2p/src/peer/base.ts

```diff
@@ -136,12 +138,17 @@ export class Peer extends EventEmitter {
 			this._resetProductivity();
 		}, DEFAULT_PRODUCTIVITY_RESET_INTERVAL);
 		this._serverNodeInfo = peerConfig.serverNodeInfo;
+		this._protocolRCPCounter = 0;
+		this._protocolRCPEvents = new Set([
+			REMOTE_EVENT_RPC_GET_NODE_INFO,
+			REMOTE_EVENT_RPC_GET_PEERS_LIST,
```

Comment:
can we rename this `REMOTE_EVENT_RPC_GET_PEERS_LIST` to `REMOTE_EVENT_RPC_GET_PEER_LIST`, because `peer list` is already list, `peers list` is weird